src/ontology/nbo-edit.owl

```diff
@@ -5113,6 +5113,7 @@ AnnotationAssertion(oboInOwl:created_by obo:NBO_0000415 "George Gkoutos")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000415 "blink reflex")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000415 "blinking")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000415 "behavior_ontology")
+AnnotationAssertion(Annotation(dc:contributor "0000-0002-4277-2151") oboInOwl:xref obo:NBO_0000415 "GO:0060082")
```

Comment:
```suggestion
AnnotationAssertion(Annotation(dc:contributor "http://orcid.org/0000-0002-4277-2151") oboInOwl:hasDbXref obo:NBO_0000415 "GO:0060082")
```

Reply:
Careful with the way you add xrefs an ORCIDs!

Reply:
I didn't know either was wrong. Thanks. I pressed the 'commit suggestion' button but now things seem to have stalled. Did it do anything? Do I have to go back to my original branch and make a change there myself?

Reply:
Not sure what is wanted from me. I will 'resolve conversation' in case that is what is needed???